backend/app.py

```diff
@@ -220,26 +294,18 @@ def create_machine():
         "geometry": {"type": "Point", "coordinates": location},
         "properties": properties_dict,
     }
-    # If pushing to new branch: set unique branch name
-    # branch_name = f"new_machine_{round(time.time())}"
-    new_machine_id = push_newmachine_to_github(new_machine_entry)
-
-    # Upload the image
-    if "image" not in request.files:
-        return "No image file", 400
```

Comment:
This is not executed anymore, but I think it's okay since we already check in the frontend whether there is an image